code/modules/mob/living/carbon/human/life.dm

```diff
@@ -335,7 +335,8 @@
 	if(!heart_attack)
 		return
 	else
-		losebreath += 5
+		if(losebreath < 3)

```

Comment:
ehhh, this should probably be 10 at most

just to make sure I can't completely nullify it with oxyloss meds